usr/share/rear/init/default/100_check_stale_nfs_mounts.sh

```diff
@@ -0,0 +1,11 @@
+# 100_check_stale_nfs_mounts.sh
+# Purpose is to have a simple test if we have stale NFS mount points which
+# could lead to a hangind ReaR session.
+# In case there is a stale NFS present we bail out with an error to get it fixed before
+# running ReaR again.
+# See issue https://github.com/rear/rear/issues/3109
+cat /proc/mounts | grep -i "[[:blank:]]nfs" | \
+    while read exported_fs mount_point junk ;
```

Comment:
How will this code handle mount points with space or other special chars in it? I think we should harden our code against such cases (or can we be sure that this is scenario is impossible?)

Reply:
If I'm not mistaken 'blank' only serves spaces and tabs. I never came across other special cases. Can you give me a real example please?

Reply:
I meant the word splitting of `read`, please try with a mountpoint like `/mnt/something else with $USER is going on!` and also there is an unquoted use of `$mount_point` below.

Reply:
Good point and I think we covered (hopefully) all special cases now.